rps.js

```diff
@@ -0,0 +1,23 @@
+const game = function(userChoice) {
+
+    let compChoice = Math.random();
+    if (compChoice < 0.30){
+     compChoice = "rock";
+    } else if (compChoice <= 0.70){
+     compChoice= "paper";
+    }
+    else { 
+     compChoice = "scissors" ;
+    }
+    
+    
+    if (userChoice == "rock" && compChoice == "rock" || userChoice == "paper" && compChoice == "paper" || userChoice == "scissors" && compChoice == "scissors") {
+    return "Tie"; }
```

Comment:
in case of a **Tie** you can check if `userChoice` is equal to  `compChoice` , that will make you code shorter.
like this:
```
if (userChoice === compChoice) {
  return "Tie";
}
```